server/src/hardware/decoder/decoderfunction.cpp

```diff
@@ -46,10 +46,15 @@ DecoderFunction::DecoderFunction(Decoder& decoder, uint8_t _number) :
     }},
   function{this, "function", DecoderFunctionFunction::Generic, PropertyFlags::ReadWrite | PropertyFlags::Store},
   value{this, "value", false, PropertyFlags::ReadWrite | PropertyFlags::StoreState,
-    [this](bool /*newValue*/)
+    [this](bool newValue)
     {
+      if(timeoutSeconds.value() > 0 && newValue)
+        m_scheduledTimeout = std::chrono::steady_clock::now() + std::chrono::seconds(timeoutSeconds.value());
+      else
+        m_scheduledTimeout = {};
       m_decoder.changed(DecoderChangeFlags::FunctionValue, number);
-    }}
+    }},
+  timeoutSeconds{this, "timeout_seconds", 0, PropertyFlags::ReadWrite | PropertyFlags::Store}
```

Comment:
Should we change it to `ms` instead of `s`, e.g. 100ms steps? Maybe 1s steps is to big.

Reply:
How do we achieve 100ms steps? Because 1ms step is too narrow. I've also noticed there is up to half second latency on turning off function so it would not respect milliseconds

Reply:
rounding to 100ms can be done using the Properties `onSet` callback and we should add a `Step` attribute, we can set it to 100, the the client can take that into account.

Reply:
Done.